docs/guides/user/cli-client.md

```diff
@@ -135,6 +135,78 @@ This example returns the field name `domain` and then do a simple sort and uniq.
 timesketch search -q "foobar" --return-fields domain | sort | uniq
 ```
 
+## Sketch
+
+### List all sketches
+
+To list all sketches you have access to:
+
+```bash
+timesketch sketch list
+```
+
+### Get description for one sketch
+
+Getting information about a sketch can be helpful in various situations.
+
+```bash
+timesketch --sketch 2 --output-format text sketch describe
+Name: asdasd
+Description: None
+Status: new
+```
+
+You can also get all stored information about a sketch with running:
+```bash
+timesketch --sketch 2 --output-format json sketch describe
+```
+
+This will give you something like:
+
+```json
+{'id': 2, 'api': <timesketch_api_client.client.TimesketchApi object at 0x7f72701461a0>, '_archived': None, '_sketch_name': 'asdasd', 'resource_uri': 'sketches/2', 'resource_data':
```

Comment:
consider pretty printing this with indentation, etc.
Also, this is not valid JSON and won't be able to get parsed by tools like `jq`. You might want to consider removing the `'api'` key and translating `None` to `false` or something similar.

Reply:
ok made it nicer, but I rather not remove any elements here, as I expect this the most raw and complete thing mostly for developers..

WDYT?

Reply:
I don't agree that is is for developers - the CLI client is for "normal" users primarily and not developers. Any json output should be parsable by jq IMO.